etl/app/server/s3.js

```diff
@@ -12,6 +12,8 @@ const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
 const environment = getEnvironment();
 
+const defaultTimeout = 60000;
```

Comment:
Can you move this setting to the `etl\app\content-private\config.json` file, so it gets stored in S3? Maybe name it something like `webServiceTimeout`.